ntbs-build/openshift-pipelines/image-streams/ntbs-image-stream.yml

```diff
@@ -0,0 +1,41 @@
+kind: ImageStream
```

Comment:
The `ImageStream` is a cache of the docker images that are used to run the tasks, so that they don't need to download a new image every time.